### ISSUE TITLE
Custom nameplate error

### ISSUE THREAD

**Issue by Talimar** [Backupiseasy/ThreatPlates#555]
Got this after typing in the name of npc:

1x bad argument #1 to '?' (Usage: local spellInfo = C_Spell.GetSpellInfo(spellIdentifier))
[string "=[C]"]: ?
[string "@TidyPlates_ThreatPlates/Options.lua"]:7939: in function <TidyPlates_ThreatPlates/Options.lua:7934>
[string "@TidyPlates_ThreatPlates/Options.lua"]:8036: in function <TidyPlates_ThreatPlates/Options.lua:8032>
[string "@TidyPlates_ThreatPlates/Options.lua"]:8186: in function <TidyPlates_ThreatPlates/Options.lua:8184>
[string "=[C]"]: ?
[string "@AdvancedInterfaceOptions/libs/AceConfig-3.0-3/AceConfigDialog-3.0-87/AceConfigDialog-3.0.lua"]:45: in function <...nfig-3.0/AceConfigDialog-3.0/AceConfigDialog-3.0.lua:43>
[string "@AdvancedInterfaceOptions/libs/AceConfig-3.0-3/AceConfigDialog-3.0-87/AceConfigDialog-3.0.lua"]:839: in function <...nfig-3.0/AceConfigDialog-3.0/AceConfigDialog-3.0.lua:661>
[string "=[C]"]: ?
[string "@AdvancedInterfaceOptions/libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:66: in function <...ancedInterfaceOptions/libs/AceGUI-3.0/AceGUI-3.0.lua:64>
[string "@AdvancedInterfaceOptions/libs/AceGUI-3.0-41/AceGUI-3.0.lua"]:300: in function `Fire'
[string "@AdvancedInterfaceOptions/libs/AceGUI-3.0-41/widgets/AceGUIWidget-EditBox.lua"]:70: in function <...ons/libs/AceGUI-3.0/widgets/AceGUIWidget-EditBox.lua:67>

Locals:
(*temporary) = "bad argument #1 to '?' (Usage: local spellInfo = C_Spell.GetSpellInfo(spellIdentifier))"

**Comment by Backupiseasy** [comment]
This should be fixed in the newest beta version for 12.0.10 - it should happen when you enter a custom nameplate name, but no NPC name or ID, i.e., this field is empty.

You write above that you entered a NPC name, but I myself always confuse both input fields (name of custom nameplate and name of NPC), so it could also be this error.

**Comment by Talimar** [comment]
Yeah that was it. I just looked at another example and it also had the npc field empty so I just entered the name.

**Comment by Backupiseasy** [comment]
This is now fixed in version 12.0.10.
